src/mon/OSDMonitor.cc

```diff
@@ -1073,14 +1130,17 @@ void OSDMonitor::prime_pg_temp(
   const OSDMap& next,
   pg_t pgid)
 {
-  if (mon->monmap->get_required_features().contains_all(
+  if (mon->monmap->get_required_features().empty()) {
+    dout(10) << __func__ << ": quorum not formed" << dendl;
```

Comment:
It's trying not to read the data if we don't expect it to be valid. I'm not sure if that's necessary or useful — I think I said elsewhere that it seemed weird for us to be invoking any of these functions while not active...